package.json

```diff
@@ -12,9 +12,12 @@
   "dependencies": {
     "@types/canvasjs": "^1.9.11",
     "canvasjs": "^1.8.3",
+    "moment": "^2.30.1",
+    "moment-timezone": "^0.5.45",
     "normalize.css": "^8.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-moment": "^1.1.3",
```

Comment:
That's probably an extra dependency. Please remove if not needed.

Reply:
Done)